<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NUi::CMainWindow</class>
 <widget class="QMainWindow" name="NMediaManager::NUi::CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/mediamanager.png</normaloff>:/resources/mediamanager.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="dirLabel">
        <property name="text">
         <string>Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NSABUtils::CDelayComboBox" name="directory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <property name="placeholderText">
         <string>&lt;directory&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string>File Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NSABUtils::CDelayComboBox" name="fileName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <property name="placeholderText">
         <string>&lt;BIF File&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionLoad"/>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionLoad"/>
   <addaction name="actionRun"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../SABUtils/resources/SABUtils.qrc">
     <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
   </property>
   <property name="text">
    <string>Set &amp;Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../SABUtils/resources/SABUtils.qrc">
     <normaloff>:/SABUtilsResources/run.png</normaloff>:/SABUtilsResources/run.png</iconset>
   </property>
   <property name="text">
    <string>Run...</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../SABUtils/resources/SABUtils.qrc">
     <normaloff>:/SABUtilsResources/load.png</normaloff>:/SABUtilsResources/load.png</iconset>
   </property>
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NSABUtils::CDelayComboBox</class>
   <extends>QComboBox</extends>
   <header>SABUtils/DelayComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileName</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>directory</tabstop>
 </tabstops>
 <resources>
  <include location="resources/application.qrc"/>
  <include location="../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>NMediaManager::NUi::CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
